src/store/editorMigrations.ts

```diff
@@ -55,6 +58,7 @@ export const migrations: MigrationManifest = {
   5: (state: EditorStateV4 & PersistedState) => migrateEditorStateV4(state),
   6: (state: EditorStateV5 & PersistedState) => migrateEditorStateV5(state),
   7: (state: EditorStateV6 & PersistedState) => migrateEditorStateV6(state),
+  8: (state: EditorStateV8 & PersistedState) => migrateEditorStateV9(state),
```

Comment:
We're seemingly missing a migration entry here?

Reply:
No, your naming is just invalid. 
```suggestion
  8: (state: EditorStateV7 & PersistedState) => migrateEditorStateV7(state),
```

Reply:
You're migrating EditorStateV7 to the new current EditorStateV8 (there is no EditorStateV9 in your pageEditorTypes)

Reply:
I think our numbering might be off. The original types are the following:

```
// Instead of maintaining old enums, just clearing data panel state on migration, see migrateEditorStateV5
export type EditorStateV7 = EditorStateV6;

export type EditorState = Except<
  EditorStateV7,
  "modComponentFormStates" | "deletedModComponentFormStatesByModId"
> & {
  modComponentFormStates: ModComponentFormState[];
  deletedModComponentFormStatesByModId: Record<string, ModComponentFormState[]>;
};
```

To me, that second type looks like a V8, we just never assigned it to a variable called EditorStateV8. Or what's going on with the types there?

Or is it some quirk with the relationship between  ModComponentFormState and BaseFormStateV5? To the final type definition should always be the latest version but with BaseFormStateVX swapped out for ModComponentFormState

Reply:
Ah I see your confusion. We always export `FooState = FooStateV<latest>` so that we don't have to update everywhere in the code that uses `FooState`. Same with the underlying `ModComponentFormState[]` vs `BaseFormStateV<latest>[]`.

It's not a V8, it's an alias

Reply:
And yes, Ben and I realize the `Except<...>` portion is very confusing. We want to make it easier to grok, but there's test/test factory ramifications that make it not straightforward, so we've been punting on it

Reply:
Gotcha, I think I understand now. I've added some comments to explain what's going on in those 2 places for people writing migrations in the future